simpleocr/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Classifiers
+from simpleocr.classification import KNNClassifier
+# Files
+from simpleocr.files import ImageFile
+# Grounders
+from simpleocr.grounding import TerminalGrounder, TextGrounder, UserGrounder
+# Improver functions
+from simpleocr.improver import enhance_image, crop_image, imagefile_to_pillow
+# OCR functions
+from simpleocr.ocr import reconstruct_chars, show_differences, OCR
+# Segmenters
+from simpleocr.segmentation import RawContourSegmenter, ContourSegmenter
+# Extraction
+from simpleocr.feature_extraction import FeatureExtractor, SimpleFeatureExtractor
+
```

Comment:
Sounds good :) We'll probably want to include more things here, such as OCR, but we can do that over time